modules/imaging_browser/php/queryengine.class.inc

```diff
@@ -171,9 +180,11 @@ class QueryEngine extends \LORIS\Data\Query\SQLQueryEngine
         if ($item instanceof QCDictionaryItem) {
             $modality = $item->getModality();
 
+            $exploded = explode("_", $item->getName());
+            $field = $exploded[count($exploded)-1];
             $this->addTable('LEFT JOIN session s ON (s.CandID=c.CandID)');
             $this->addTable("LEFT JOIN files ON (s.ID=files.SessionID)");
-            return "(SELECT QCStatus FROM files_qcstatus
+            return "(SELECT $field FROM files_qcstatus
```

Comment:
Note from meeting:
This is where it maps a dictionary item to its source in SQL, since no part of the existing dictionary item directly maps to an SQL header name, Rida did some magic here to extract it from the item name